### DISCUSSION TITLE
srsRAN performance on Intel vs AMD

### DISCUSSION THREAD

**Discussion by JonPichel** [srsran/srsRAN_4G#1227] [Q&A] [answered]
Hello everyone,

I've been looking at the source code of the project and I found that you use MKL (https://github.com/srsran/srsRAN_4G/blob/master/cmake/modules/FindMKL.cmake). My question is if this translates into a noticeable difference in performance when using AMD vs using Intel CPUs. Should I prefer an Intel CPU over an AMD CPU with similar characteristics when choosing a machine to run this software?

**Comment by andrepuschmann** [comment, marked as answer]
In general, there is no significant performance improvement using KML. In fact `libfftw` performs very well on both platforms so there is no reason to worry.
